docs/anomaly-detection/QuickStart.md

```diff
@@ -0,0 +1,102 @@
+---
+sort: 1
+weight: 1
+title: VictoriaMetrics Anomaly Detection Quick Start
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Quick Start
+aliases:
+- /anomaly-detection/QuickStart.html
+---
+
+# `vmanomaly` Quick Start
+
+It is recommended to read [README](https://docs.victoriametrics.com//vmanomaly.html/)
```

Comment:
there is a typo in `https://docs.victoriametrics.com//vmanomaly.html/` (double //), also, vmanomaly.html (alias, as the page was moved to [/overview](https://docs.victoriametrics.com/anomaly-detection/overview/) redirects exactly to https://docs.victoriametrics.com/anomaly-detection/ , so _you can use this full link instead_